.drone.yml

```diff
@@ -45,91 +48,99 @@ pipeline:
       - php occ app:enable files_antivirus
       - php occ app:enable testing
       - php occ app:list
-      - php occ config:system:set trusted_domains 1 --value=server
+      - php occ config:system:set trusted_domains 1 --value=owncloud
```

Comment:
Any particular reason for the change in container naming?

Reply:
just to do it the same as many of the other app repos

Reply:
If it is the same in every repo, then it is so easy to cut-paste blocks of code into ``.drone.yml`` files and minimize the amount of thinking needed about what little things need to be adjust and where.

Of course, we can decide that ``server`` is going to be the standard name for the ownCloud server service, and adjust the apps that currently call it ``owncloud``

Reply:
Seems fine to have the same naming everywhere - whichever (owncloud vs server) is fine for me. 

I am a bit concerned about the copy/cut + paste part - hopefully we can agree that change to CI should be done by trying to understand how it works. Otherwise things will get copy+pasted around without people knowing what actually is going on -> false sense of security

Reply:
I agree, it is good that people understand what they are doing!

But also, if they do it a bit different in each repo then we end up with long reviews, as the reviewers need to ask why. If we make things the same, then [hopefully|maybe some day] review of this sort of thing will be simple.

(and course if there was a way for an app to "suck in" lots of this "standard" CI script infrastructure, that would be even better - no cut-pasting of blocks of script)